src/frontend/data/appConfigs.ts

```diff
@@ -0,0 +1,12 @@
+import { appNames, flags } from 'types/AppData';
+
+/**
+ * Configure feature flag to control display of declared application
+ * Feature flag is optional here, declared app without configuration is displayed by default
+ * When a flag is specified :
+ *  - if the associated flag is disable, app will not be loaded and displayed
+ *  - if the associated flag is enabled, app will be loaded and displayed
+ */
+export const appConfigs: { [key in appNames]?: flags } = {
```

Comment:
`appConfigs` is not relevant IMO. I expect to find configuration related to an app and it's not the case.
Maybe having a structure like this is more relevant ?

```
{
  bbb: {
    flag: 'BBB',
  }
}
```

And this structure will evolve in the time with future information we will need ?

Reply:
Yep, I can type it like `{ [key in appNames]?: { flag?: flags, } }`

I'm just wondering if this is not "over engineering" since we will use only the flag for now (as far as I know XD).
You tell me, it wont impact too much this PR and I can do it pretty fast

Reply:
Or just rename the variable with a more relevant name, you choose.

Reply:
> There are only two hard things in Computer Science: cache invalidation and naming things.
>
> -- Phil Karlton

Reply:
My 2 cents here, maybe we could keep this structure, and remame with `appFlags` ?

Reply:
arf, too late